src/database/sqlite/sqlite_aclk.c

```diff
@@ -957,11 +946,10 @@ void destroy_aclk_config(RRDHOST *host)
     if (!host || !(ahc = host->aclk_config))
         return;
 
-    if (ahc->timer_initialized) {
-        if (uv_is_active((uv_handle_t *)&ahc->timer))
-            uv_timer_stop(&ahc->timer);
-        uv_close((uv_handle_t *)&ahc->timer, NULL);
-    }
+//    if (ahc->timer_initialized) {
```

Comment:
Is there a possibility to return this code in a near future, for we keep it commented?

Reply:
deleted for now